how-to-build.md

```diff
@@ -146,13 +146,23 @@ These are the available variables:
 
 ## Running [app] tests against a local MongoDB Stitch
 
+Due to MongoDB security policies, running baas requires company issued AWS account credentials.
+If you do not have these, request some from Tim Sedgwick on the cloud team, or reach out to #realm-core.
```

Comment:
Let's not put Tim's name as the point of contact in public documentation. Just reach out to #realm-core.